main.go

```diff
@@ -209,6 +215,13 @@ func collector(fh *os.File) {
 		statistics.Add(*procstats)
 	}
 
+	memory, err := mem.VirtualMemory()
+	if err != nil {
+		log.Printf("Unable to collect system memory statistics\n")
+		return
+	}
+	statistics.AvailableMemory = memory.Available
```

Comment:
👍 I wonder whether it makes sense to inject `UsedPercent` too? Otherwise we'll have to compute this in BigQuery? Or maybe we don't care? We just look at the graphs and know that we have X GB RAM extra and we aim to level that to zero.

Reply:
Hmm, either `Used` or `UsedPercent`, definitely. I'll add `UsedPercent` for now